cdap-proto/src/main/java/co/cask/cdap/proto/audit/AuditMessage.java

```diff
@@ -27,7 +27,8 @@
  */
 @Beta
 public class AuditMessage {
-  private final int version = 1;
+  // version 1 has consisted of EntityId and version 2 consists of MetadataEntity
+  private final int version = 2;
```

Comment:
I assume we are changing this back?

Reply:
Yes this will be changed back in the backward compatibility change.

Reply:
After the change in approach we have decided to emit MetadataEntity in AuditMessage so this change will be needed. I had the change to be able to read old AuditMessage for backward compatibility.

As you pointed out during our in-person conversation that this will require changes in Tracker app to understand the new message. I will look in to it and figure out the changes required there. I have added a comment in the main jira to track this.